README.rst

```diff
@@ -13,7 +13,11 @@ hnn-core
 	:target: https://codecov.io/gh/jonescompneurolab/hnn-core
 	:alt: Test coverage
 
-This is a leaner and cleaner version of the code based off the `HNN repository <https://github.com/jonescompneurolab/hnn>`_. However, a Graphical User Interface is not supported at the moment in this repository.
+.. image:: https://user-images.githubusercontent.com/15852194/115123685-2df1ef00-9f8c-11eb-8f3b-663486466193.png
```

Comment:
Discussion point for us, what do we want to do in terms of naming? If the GUI is part of the `hnn-core` repo, then the naming scheme of GUI/core is a bit confusing. 

Maybe it's sufficient to just be really clear in the website documentation.

Reply:
This comes to mind because putting a picture of the GUI on the README signals that this is no longer just the "core" functionality of HNN

Reply:
Is our end-goal to have the new GUI be part of a separate repo from hnn-core?

Reply:
I think the only immediate concern here was to determine if and how much we should be modifying the README.rst if we have no intention of making the GUI a permanent part of hnn-core. In the spirit of moving on from this discussion, I propose that we avoid making GUI-related changes to README.rst and whats_new.rst so that the GUI remains an internal affair until we decide how it will be distributed to the general public later.

Reply:
okay got it. Let's remove this image link for now.